config/migrations/2023/20230322170100-fix-vendor-management-mock-user-roles.sparql

```diff
@@ -0,0 +1,13 @@
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
```

Comment:
@Windvis cool!  I am not sure this migration is needed. :-)

 Yeah totally forgot that we modify bestuurseenheden that are in public and read only.
BUT I THINK you can use similar construct https://github.com/lblod/app-digitaal-loket/blob/master/config/authorization/config.ex#L326
for this https://github.com/lblod/app-digitaal-loket/blob/master/config/authorization/config.ex#L340 for the public graph and that specific predicate

Reply:
I added them because that user seemed to have all other roles as well. I can take a look at the mu-auth config but that thing is still an enigma to me at the moment.

Reply:
i will do it